codespell_lib/data/dictionary.txt

```diff
@@ -35845,6 +35848,7 @@ larvays->larvae
 larvy->larvae
 laso->also, lasso,
 lasonya->lasagna
+lasr->laser, last,
```

Comment:
Could also be `lash` and `lair`, but I couldn't find any such typo, so let's keep it as is.